runner/repositories_test.go

```diff
@@ -51,6 +54,22 @@ type RepoTestSuite struct {
 	Runner   *Runner
 }
 
+func (s *RepoTestSuite) EqualReposByName(expected map[string]params.Repository, actual []params.Repository) {
```

Comment:
This doesn't need to be exported if you don't want to call this function from another package. Functions that start with a capital letter are exported. If you'll just use it in this package it can be renamed to ```equalReposByName()```

Reply:
It's easier to compare two sorted slices instead of a map and a slice. To sort a slice you can use:

https://pkg.go.dev/sort#Slice

Reply:
Thank you for the suggestion. I implemented this and I updated the PR.